src/django_components/dependencies.py

```diff
@@ -1036,3 +1038,66 @@ def _process_response(self, response: HttpResponse) -> HttpResponse:
             response.content = render_dependencies(response.content, type="document")
 
         return response
+
+
+#########################################################
+# 6. Template tags
+#########################################################
+
+
+def _component_dependencies(type: Literal["js", "css"]) -> SafeString:
```

Comment:
The `{% component_js_dependencies %}` and  `{% component_css_dependencies %}` were moved to this file for cosistency.